containers/rust-postgres/.devcontainer/docker-compose.yml

```
@@ -0,0 +1,54 @@
+version: '3.8'
+
+volumes:
+  postgres-data:
+
+services:
+  app:
+    container_name: rust
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        # Use the VARIANT arg to pick a Debian OS version: buster, bullseye
+        # Use bullseye when on local on arm64/Apple Silicon.
+        VARIANT:  buster
+        # Options
+        NODE_VERSION: "lts/*"
+    env_file:
+        # Ensure that the variables in .env match the same variables in devcontainer.json
+        - .env
+    # Security Opt and cap_add.
```

Let's make this more clear. Something like:

  
```suggestion
     # Security Opt and cap_add allow for C++ based debuggers to work. 
```